### DISCUSSION TITLE
Changes in docs: Asking about the thoughts behind removing the quirks

### DISCUSSION THREAD

**Discussion by FleetAdmiralJakob** [nativewind/nativewind#1016] [General]
In the v2 docs, there is a section called Quirks. (https://www.nativewind.dev/core-concepts/quirks) This section was removed in the v4 docs. So my question is now: What has changed and how did these issues get resolved in v4?

I'm especially interested in the pixel density vs pixel case.

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
